push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -208,10 +208,10 @@ private Date createFutureDateBasedOnTTL(int ttl) {
 
         // no TTL was specified on the payload, we use the MAX Default from the APNs library:
         if (ttl == -1) {
-            return new Date(System.currentTimeMillis() + EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
+            return new Date(EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
```

Comment:
@lfryc `EnhancedApnsNotification.MAXIMUM_EXPIRY` is just a constant it's not meant to be turned into a date. If we remove `System.currentTimeMillis()` the result will be: `Sun Jan 25 21:31:23 CET 1970` so either we leave it as it was or we change it to `new Date(Long.MAX_VALUE)` both options are fine with me, but the change as it it proposed here is wrong